doc/usermanual/darkroom/modules/effects/group_effects.xml

```diff
@@ -1,35 +1,44 @@
-<sect2 status="final" id="effect_group">
+<sect2 status="final" id="effects_group">
```

Comment:
Don't we need to update the integrated help (when clicking on ?) as links have changed? Probably true for many other changes here I think.

Reply:
oh... forgot that... But yes, there should be many links to change (at least all the iops...) I'll have a look !

Reply:
hmm... so it seems I have *tons* of links to change in `usermanual_url.c ` (basically each iop link as changed)
problem : once they are changed, they will point to invalid urls until we release the manual... should I do this anyway ? or should we postponed it ?

Reply:
done. and in the same time I've fixed lot of invalid urls or missing ones. Surprising that no user raise issues... Maybe a sign that this feature is underutilized ?